.github/actions/rust-build-release/tests/test_manifest_input_step.py

```
@@ -21,6 +29,76 @@ def _find_step(steps: list[dict[str, object]], name: str) -> dict[str, object]:
+    bash = shutil.which("bash")
+    if bash is None:  # pragma: no cover - bash is present on supported runners
+        pytest.skip("bash not found on PATH")
+    tmp_path.mkdir(parents=True, exist_ok=True)
+    github_env = tmp_path / "github-env"
+    github_env.touch()
+    env = {key: value for key, value in os.environ.items() if key != "RUSTFLAGS"}
+    env["GITHUB_ENV"] = github_env.as_posix()
+    env["RBR_RUSTFLAGS"] = rustflags
+    if inherited is not None:
+        env["RUSTFLAGS"] = inherited
+    result = subprocess.run(  # noqa: S603,TID251 - exercise the bash fragment.
```

Ruff (0.16.0)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)